content/rust/content/structs/structs.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'Structs'
+Description: 'Rust structs are custom data structures used to group related data fields together.'
+Subjects:
+  - 'Data Science'
+  - 'Web Development'
+
+Tags:
+  - 'Comments'
+  - 'Documentation'
+  - 'Structure'
+CatalogContent:
+  - 'rust-for-programmers'
+---
+
+In Rust, a struct (short for "structure") is a composite data type used to group together multiple fields under a single name. Each field in a struct can have its own data type, allowing it to represent complex data structures. Structs are defined using the 'struct' keyword.
+
+## Defining Struct
+
+```rust
+struct Person {
+    name: Boric,
+    age: 22,
+    address: String,
+}
+```
+In this `Person` struct, there are three compartments (fields): name, age, and address. Each field has a specific data type (like `String`) that defines the kind of data it can store.
```

Comment:
````suggestion
```

In this `Person` struct, there are three compartments (fields): name, age, and address. Each field has a specific data type (like `String`) that defines the kind of data it can store.
````